src/styles/actor/_index.scss

```diff
@@ -225,6 +225,10 @@
             grid-area: nav;
         }
 
+        button {
+            min-height: unset;
+        }
```

Comment:
?

Reply:
Foundry v13 adds min-height: var(--button-size); to the global styling. You can see the biggest break in the strikes list on a versatile weapon.

The more appropriate place might be reset.css, and I'm considering doing that if I see it fixes issues on other sheets.

Reply:
Just set `--button-size` instead of a height

Reply:
Should we be setting theme variables on a per element basis? I actually kinda regret doing that on the party sheet tags, since theme variables are a thing that should be set more globally to the sheet itself I think for best compatibility.

Reply:
It's not a theme variable (light and dark don't have different values).

Reply:
A theme variable isn't necessarily for light vs dark, but also for theoretical third party themes in the future. We have theme variables in pf2e and we don't have a light vs dark.

Granted, button-size is kinda awkward to change globally, so even that's an odd case.

Reply:
I super duper promise that `--button-size` has nothing to do with themes.

Reply:
I could set --button-size: auto in the reset.scss? I'd need to verify tomorrow to make sure everything is fine though and ensure its in a reset layer. If I set --button-size on each button, It'll take me a while as I comb over every button. Either or I'll have to tackle it tomorrow.

Reply:
I don't think you need to do anything with a reset file: just update the buttons that need it.